Vagrantfile

```diff
@@ -13,10 +13,12 @@ apt-get update -qq
 apt-get install -y -qq --no-install-recommends chrony
 EOS
 
```

Comment:
I'm wondering if there is a way we can put in some logical if-then conditions if the user wants to use KVM or the standard Virtualbox. Virtualbox might have performance issues, but for a developer who wants to quickly test Promenade, it might not be a bad idea to support both.